examples/somersaultecu.py

```diff
@@ -767,7 +767,7 @@ class SomersaultSID(IntEnum):
                     long_name=None,
                     semantic=None,
                     description=None,
-                    request_byte_position=3,
+                    request_byte_position=0,
```

Comment:
why?

Reply:
because using byte position 3 is outside of the constant prefix of the request and this breaks the [`._prefix_tree`](https://github.com/mercedes-benz/odxtools/blob/main/odxtools/diaglayer.py#L1030) code if the `MatchingRequest` parameter is made to be strict in the sense that it must encode something. (I have a patch for the latter and I did not want to open the can of worms of thinking about how to represent non-existing matching request data during encoding. also, I think this is a nice demonstration why one might want to modify this database dynamically instead of just copying `somersaultecu.py` and fiddling with the copy...)